src/include/cp_control_point_gen.h

```diff
@@ -0,0 +1,61 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+/* cp_control_point_gen.h: Generated declarations for control points. */
+/* In the future this file will be generated. Until then this file must be edited when a new control
+ * point is created. */
+
+#ifdef HAVE_CONTROL_POINT
+
+/*
+ * The name below is for a per connection control point named "Example control point".
+ *
+ * Each per connection control point has:
+ * - Per connection control point ID (Could be generated):
+ * WT_CONN_CONTROL_POINT_ID_EXAMPLE_CONTROL_POINT.
+ *
+ * The name below is for a per session control point named "Example control point2".
+ *
+ * Each per session control point has:
+ * - Per session control point ID (Could be generated):
+ * WT_CONN_CONTROL_POINT_ID_EXAMPLE_CONTROL_POINT2.
+ */
+
+#include "control_points.h"
+
+/*
+ * Per connection control point IDs.
+ */
+#define WT_CONN_CONTROL_POINT_ID_MAIN_START_PRINTING ((wt_control_point_id_t)0)
```

Comment:
As IDs for are the example control points (in ex_control_points.c), I think these should probably live with the example code. There's no need to change this for now, but we should discuss the best solution for this.